web_timeline/static/lib/vis/vis.css

```diff
@@ -0,0 +1,810 @@
+.vis .overlay {
```

Comment:
I agree with @pedrobaeza. In Odoo the 'implicit' convention is to include the libraries in the source code. Moreover, npm requires that you have acces to internet from your server and it's not always the case.